config/config.js

```diff
@@ -1,15 +1,15 @@
-const requireEnv = require('require-environment-variables');
-
-const dbId = process.env.DB_ID || 'services';
-const collectionId = process.env.DB_COLLECTION_ID || 'services';
-const dbUrl = `dbs/${dbId}`;
-
-requireEnv(['DB_PRIMARY_KEY']);
+const db = process.env.MONGODB_DB || 'services';
+const collection = process.env.MONGODB_COLLECTION || 'pharmacies';
+const host = process.env.MONGODB_HOST || 'mongo';
+const port = process.env.MONGODB_PORT || 27017;
+const connectionString = `mongodb://${host}:${port}/${db}`;
 
 module.exports = {
-  docDB: {
-    endpoint: process.env.DB_ENDPOINT || 'https://connecting-to-services.documents.azure.com:443/',
-    primaryKey: process.env.DB_PRIMARY_KEY,
-    collectionUrl: `${dbUrl}/colls/${collectionId}`,
+  mongodb: {
```

Comment:
Do all of these properties need to be exported? From what I can see only `collection` and `connectionString` are used within the app. The others are only used in the test.